openadr-vtn/src/data_source/postgres/mod.rs

```diff
@@ -53,6 +53,11 @@ impl DataSource for PostgresStorage {
     fn auth(&self) -> Arc<dyn AuthSource> {
         Arc::<PgAuthSource>::new(self.db.clone().into())
     }
+
+    ///Verify the connection pool is open and has at least one connection
```

Comment:
I could not find any official "rule" for that, but to me, it seems that (doc) comments typically have a space between the slash and first letter, i.e., `/// Verifiy ...` instead of `///Verify ...`. It would be great to keep it consistent in this repository.